conf/eslint.json

```diff
@@ -231,6 +231,7 @@
         "strict": "off",
         "symbol-description": "off",
         "template-curly-spacing": "off",
+        "template-tag-spacing": "off",
```

Comment:
Good idea on the dog fooding! But agreed with @platinumazure that it should probably be a separate PR